tests/test_elections_2.py

```diff
@@ -0,0 +1,185 @@
+import votekit.election_types as et
+from votekit.ballot import Ballot
+from votekit.profile import PreferenceProfile
+from votekit.election_state import ElectionState
+from votekit.cvr_loaders import rank_column_csv
+
+from fractions import Fraction
+from pathlib import Path
+
+
+BASE_DIR = Path(__file__).resolve().parent
+DATA_DIR = BASE_DIR / "data"
+TEST_PROFILE = rank_column_csv(DATA_DIR / "test_election_B.csv")
+condo_ballot_list = [
+    Ballot(
+        id=None, ranking=[{"C"}, {"A"}, {"D"}, {"E"}, {"B"}], weight=Fraction(10, 1)
+    ),
+    Ballot(
+        id=None, ranking=[{"A"}, {"B"}, {"C"}, {"E"}, {"D"}], weight=Fraction(10, 1)
+    ),
+    Ballot(
+        id=None, ranking=[{"D"}, {"A"}, {"B"}, {"C"}, {"E"}], weight=Fraction(10, 1)
+    ),
+]
+CONDO_TEST_PROFILE = PreferenceProfile(ballots=condo_ballot_list)
+
+
+def equal_electionstates(state1, state2):
+    print(state1.get_all_winners(), state1.get_all_eliminated(), state1.get_rankings())
+    print(state2.get_all_winners(), state2.get_all_eliminated(), state2.get_rankings())
+
+    return (

```

Comment:
output for errors is pretty tough to debug, probably why you have the print statements in here

I'd recommend instead using multiple assert statements, `pytest` will highlight the errors better if you assert winners1=winners2 then assert elim1 == elim2, etc.